src/metatrain/utils/data/target_info.py

```diff
@@ -47,8 +48,14 @@ def gradients(self) -> List[str]:
 
     @property
     def per_atom(self) -> bool:
-        """Whether the target is per atom."""
-        return "atom" in self.layout.block(0).samples.names
+        """Whether the target is per atom. Also applies to per atom pair quantities."""
+        return "atom" in self.layout.block(0).samples.names or (
+            "first_atom" in self.layout.block(0).samples.names and
+            "second_atom" in self.layout.block(0).samples.names and
+            "cell_shift_a" in self.layout.block(0).samples.names and
+            "cell_shift_b" in self.layout.block(0).samples.names and
+            "cell_shift_c" in self.layout.block(0).samples.names
+        )
```

Comment:
I don't find it very intuitive to return `True` from this function if the target is per-pair. I would create a new `@property` that's named `per_pair`

Reply:
I tried this, but the problem then comes when defining a `ModelOutput(per_atom=per_atom)`. This is another discussion though (to be had with @Luthaf too), and perhaps for now I could do this as you say and just use `ModelOutput(per_atom=per_atom or per_pair)`

Reply:
This is kept as is for now. "per_atom" is used as a temporary workaround in lieu of there being a more flexible option in `ModelOutput`